examples/preconditioner-export/doc/intro.dox

```diff
@@ -0,0 +1,5 @@
+<a name="Intro"></a>
+<h1>Introduction</h1>
+
+<h3> About the example </h3>
+
```

Comment:
I guess there should be something in here?

Reply:
```suggestion

This example shows how to explicitly generate and store preconditioners for a given matrix.
It can also be used to inspect and debug the preconditioner generation.
```